.travis.yml

```diff
@@ -26,6 +26,8 @@ matrix:
         - BACKEND=OpenBLAS
         - USE_VTK=ON
         - ENABLE_PYTHON=ON
+        - CMAKE_VERSION=3.16.2
+        - SWIG_VERSION=4.0.1
```

Comment:
is this the minimal version we support now?

Reply:
For swig yes, definitely. swig 3 has a bug which was never corrected in the 3.x series.
For cmake as I said above, versions > 3.12 might be working.... I'm almost sure that in the debug process I used a 3.14 (before switching to 3.16) and it had the same symptoms 3.16 had before I found the real bug.... So going back to 3.14 is certainly doable.